### ISSUE TITLE
Cannot find module '/node_modules/tailwindcss/plugin' when working with @formkit/tailwindcss

### ISSUE THREAD

**Issue by nklsw** [formkit/formkit#142]
### Reproduction

https://stackblitz.com/edit/formkit-nuxt-tailwind

### Describe the bug

I'm trying to use @formkit/tailwindcss with FormKit and Nuxt 3 like explained [in the docs](https://formkit.com/guides/create-a-tailwind-theme). But I get the following error when I run dev:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/projects/formkit-nuxt-tailwind/node_modules/tailwindcss/plugin' imported from /home/projects/formkit-nuxt-tailwind/node_modules/@formkit/tailwindcss/dist/index.mjs
Did you mean to import tailwindcss/plugin.js?
    at NodeError.get (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:286334)
    at __node_internal_ (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:1492)
    at new NodeError (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:2995)
    at https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:367454
    at https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:367501
    at Loader.defaultResolve [as _resolve] (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:367507)
    at Loader.resolve (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:1019940)
    at Loader.getModuleJob (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:1021996)
    at https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:6:734578
    at _0x52ceac.link (https://formkit-nuxt-tailwind.w.staticblitz.com/blitz.5bca79c80956550a0ee18f24bf5b874e18bb8626.js:15:232691) {
  code: 'ERR_MODULE_NOT_FOUND'
}
[vite dev] Error loading external "/home/projects/formkit-nuxt-tailwind/node_modules/@formkit/tailwindcss/dist/index.mjs".
  at async eval (file://./.nuxt/nitro/index.mjs:160:24)  
  at async renderMiddleware (file://./.nuxt/nitro/index.mjs:215:20)  
  at async handle (file://./node_modules/h3/dist/index.mjs:299:19)
````

I also tried using it with @nuxtjs/tailwindcss but got the same error. 
After quick research I found a [similar issue](https://github.com/tailwindlabs/tailwindcss-forms/issues/31) at @tailwindcss/forms. But none of the proposed solution fixed the problem. 

Can someone provide a working sandbox for with Nuxt and @formkit/tailwindcss? 

### Environment

• Tested on Stackblitz, macOS 12.2 and Windows 10
• @formkit/nuxt v1.0.0-beta.5

**Comment by GustavoFenilli** [comment]
For now, try to update formkit to the @next of @formkit/tailwindcss version, I think it was fixed at #134.

**Comment by nklsw** [comment]
Sorry I overlooked that this is a clear duplicate of #134
